hydra-cluster/test/Test/DirectChainSpec.hs

```diff
@@ -408,7 +407,7 @@ spec = around showLogsOnFailure $ do
             -- note, that we don't try to check whether it's failing for the
             -- right reason here (see corresponding mutation test for this).
             contestAgain `shouldThrow` \case
-              (ScriptFailedInWallet{} :: PostTxError Tx) -> True
+              (InternalWalletError{} :: PostTxError Tx) -> True
```

Comment:
Why did this exception change? The previous exception was much more concrete (judging from the note above)